pkgs/applications/office/cutemarked-ng/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qmake
+, qttools
+, qtbase
+, qtwebengine
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "CuteMarkEd-NG";
+  version = "unstable-2021-07-29";
+
+  src = fetchFromGitHub {
+    owner = "Waqar144";
+    repo = pname;
+    rev = "9431ac603cef23d6f29e51e18f1eeee156f5bfb3";
+    sha256 = "sha256-BiF63o/XbnUvDBGX2lGTb2Ht589bHcc31UcyXfPhC/c=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    qmake
+    qttools
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtwebengine
+  ];
+
+  postPatch = ''
+    substituteInPlace app/app.pro \
+      --replace '$$[QT_INSTALL_BINS]/lrelease' "lrelease"
+  '';
```

Comment:
```suggestion
```

Reply:
it's necessary.  lrelease is a tool of `qttools`, but `QT_INSTALL_BINS`   points to the path of `qtbase`

Reply:
I didn't remove it, I only moved it to a more standard location.

Reply:
> I didn't remove it, I only moved it to a more standard location.
Sorry，I misread it